Toolset/libraries/revinitialisationlibrary.livecodescript

```diff
@@ -16,18 +16,26 @@ command revInternal__LoadLibrary pLibrary
    end if
    
    // AL-2015-02-20: [[ IDE Restructure ]] Look for libraries in the correct locations.
-   set the itemDelimiter to "."
    if the environment begins with "development" then
       local tLibraryPath, tLibSuffix
+      set the itemDelimiter to "."
+      put the last item of pLibrary into tLibSuffix
       
       if not revEnvironmentIsInstalled() then
-         -- look for libraries in parent repository
-         if there is a file (revEnvironmentRepositoryPath() & "/ide-support/" & pLibrary & ".livecodescript") then
-            put revEnvironmentRepositoryPath() & "/ide-support/" & pLibrary & ".livecodescript" into tLibraryPath
+         -- look for libraries in parent development locations
+         local tRepoPath
+         put revEnvironmentRepositoryPath() into tRepoPath
+         if there is a file (tRepoPath & "/ide-support/" & pLibrary & ".livecodescript") then
+            put tRepoPath & "/ide-support/" & pLibrary & ".livecodescript" into tLibraryPath
          else 
-            put the last item of pLibrary into tLibSuffix
-            if there is a file (revEnvironmentRepositoryPath() & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript") then
-               put revEnvironmentRepositoryPath() & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript" into tLibraryPath
+            if there is a file (tRepoPath & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript") then
+               put tRepoPath & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript" into tLibraryPath
+            else
+               set the itemDelimiter to slash
+               delete the last item of tRepoPath
+               if there is a file (tRepoPath & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript") then
+                  put tRepoPath & "/extensions/script-libraries/" & tLibSuffix & slash & tLibSuffix & ".livecodescript" into tLibraryPath
+               end if
```

Comment:
Hmm... I'm not sure that adds a great deal. Another approach would be to build a list of likely locations and iterate them.